components/public-api/typescript/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+export * as experimental from "./gitpod/experimental/v1";
```

Comment:
If there was a `gitpod/experimental/v2`, how would that be exported such that the client can choose v1 or v2?

Reply:
> Let's go with this for now, but I think we're still missing the ability to import all of the following:
> 
> stable/v1
> stable/v2
> experimental/v1
> experimental/v2

mmm I don't think is a common pattern in npm package to ship multiple versions at the same time, but can think about it later :thinking: